3.2/root/usr/bin/run-redis

```diff
@@ -6,6 +6,13 @@ set -eu
 
 [ -f ${CONTAINER_SCRIPTS_PATH}/validate-variables.sh ] && source ${CONTAINER_SCRIPTS_PATH}/validate-variables.sh
 
+S2I_REDIS_CONFIG=${APP_DATA}/redis-cfg/redis.conf
```

Comment:
We may allow overriding this path, although it probably doesn't need to be documented:
```
S2I_REDIS_CONFIG=${S2I_REDIS_CONFIG:-${APP_DATA}/redis-cfg/redis.conf}
```